tests/test_crs/valid_fio.yaml

```diff
@@ -6,13 +6,13 @@ metadata:
 spec:
   cleanup: false
   workload:
-    name: fio
-    args:
-      # To disable fio, set clients to 0
-      clients: 1
-      jobname: test-write
-      bs: 4k
-      iodepth: 1
-      runtime: 12
-      rw: write
-      filesize: 1
+  fio:
```

Comment:
the indentation is not correct, it should be as how it was before. there needs to be the variable workload.name and workload.args